### DISCUSSION TITLE
Autocraft mistake

### DISCUSSION THREAD

**Discussion by FTYears** [ORelio/Minecraft-Console-Client#1971] [Q&A]
I'm sure that I use autocraft correctly, but mcc always tells me that table not found. Meanwhile I use /dig at the same position as I write in the config,ini. the bot can dig the table. That really makes me mad. Please tell me why it occures.Thanks a lot.
[AutoCraft]
# A valid autocraft config must begin with [AutoCraft]

tablelocation=7943,4,5022   # Location of the crafting table if you intended to use it. Terrain and movements must be enabled. Format: x,y,z
onfailure=abort        # What to do on crafting failure, abort or wait
# You can define multiple recipes in a single config file
# This is an example of how to define a recipe
[Recipe]
name=a         
type=table           
result=Melon   

# define slots with their deserved item
slot1=MelonSlice     
slot2=MelonSlice  
slot3=MelonSlice  
slot4=MelonSlice  
slot5=MelonSlice  
slot6=MelonSlice  
slot7=MelonSlice  
slot8=MelonSlice  
slot9=MelonSlice

**Comment by ReinforceZwei** [comment]
Maybe you could do `/useblock x y z` at your table and see if MCC opens a container.
